presto-bytecode/src/main/java/com/facebook/presto/bytecode/MethodDefinition.java

```diff
@@ -30,13 +31,16 @@
 import static com.facebook.presto.bytecode.Access.STATIC;
 import static com.facebook.presto.bytecode.Access.toAccessModifier;
 import static com.facebook.presto.bytecode.ParameterizedType.type;
+import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.collect.Iterables.transform;
 import static org.objectweb.asm.Opcodes.RETURN;
 
 @SuppressWarnings("UnusedDeclaration")
 @NotThreadSafe
 public class MethodDefinition
 {
+    private static final int MAX_ALLOWED_VARARG_ARGUMENTS = 250;
```

Comment:
as far as i know long and doubles count as 2, so you can pass the check and still hit jvm limit.

(btw. the constant name is still wrong.. but @electrum requested removing constant anyway)